po/fr.po

```diff
@@ -0,0 +1,15296 @@
+msgstr "alloc"
+
+#: src/SUMMARY.md:243
+msgid "Microcontrollers"
+msgstr "Microcontrôleurs"
+
+#: src/SUMMARY.md:244
+msgid "Raw MMIO"
+msgstr ""
+
+#: src/SUMMARY.md:245
+msgid "PACs"
+msgstr ""
+
+#: src/SUMMARY.md:246
+msgid "HAL Crates"
+msgstr ""
+
+#: src/SUMMARY.md:247
+msgid "Board Support Crates"
+msgstr ""
+
+#: src/SUMMARY.md:248
+msgid "The Type State Pattern"
+msgstr "Le modèle d'état de type"
+
+#: src/SUMMARY.md:249
+msgid "embedded-hal"
+msgstr ""
+
+#: src/SUMMARY.md:250
+msgid "probe-rs, cargo-embed"
+msgstr ""
+
+#: src/SUMMARY.md:251
+msgid "Debugging"
+msgstr "Débogage"
+
+#: src/SUMMARY.md:252 src/SUMMARY.md:270
+msgid "Other Projects"
+msgstr "Autres projets"
+
+#: src/SUMMARY.md:254
+msgid "Compass"
+msgstr "Boussole"
+
+#: src/SUMMARY.md:256
+msgid "Bare Metal Rust: Afternoon"
+msgstr "Bare Metal Rust : Après-midi"
+
+#: src/SUMMARY.md:258
+msgid "Application Processors"
+msgstr "Processeurs d'applications"
+
+#: src/SUMMARY.md:259
+msgid "Inline Assembly"
+msgstr "Assemblage en ligne"
+
+#: src/SUMMARY.md:260
+msgid "MMIO"
+msgstr "MMIO"
+
+#: src/SUMMARY.md:261
+msgid "Let's Write a UART Driver"
+msgstr "Écrivons un pilote UART"
+
+#: src/SUMMARY.md:262
+msgid "More Traits"
+msgstr "Plus de traits"
+
+#: src/SUMMARY.md:263
+msgid "A Better UART Driver"
+msgstr "Un meilleur pilote UART"
+
+#: src/SUMMARY.md:264
+msgid "Bitflags"
+msgstr "Bitflags"
+
+#: src/SUMMARY.md:265
+msgid "Multiple Registers"
+msgstr "Registres multiples"
+
+#: src/SUMMARY.md:266
+msgid "Driver"
+msgstr "Périphérique"
+
+#: src/SUMMARY.md:267 src/SUMMARY.md:269
+msgid "Using It"
+msgstr "En l'utilisant"
+
+#: src/SUMMARY.md:271
+msgid "Useful Crates"
+msgstr "Crates utiles"
+
+#: src/SUMMARY.md:272
+msgid "zerocopy"
+msgstr ""
+
+#: src/SUMMARY.md:273
+msgid "aarch64-paging"
+msgstr ""
+
+#: src/SUMMARY.md:274
+msgid "buddy_system_allocator"
+msgstr ""
+
+#: src/SUMMARY.md:275
+msgid "tinyvec"
+msgstr ""
+
+#: src/SUMMARY.md:276
+msgid "spin"
+msgstr ""
+
+#: src/SUMMARY.md:278
+msgid "vmbase"
+msgstr ""
+
+#: src/SUMMARY.md:280
+msgid "RTC Driver"
+msgstr "Périphérique RTC"
+
+#: src/SUMMARY.md:284
+msgid "Solutions"
+msgstr "Solutions"
+
+#: src/SUMMARY.md:289
+msgid "Day 1 Morning"
+msgstr "Jour 1 Matin"
+
+#: src/SUMMARY.md:290
+msgid "Day 1 Afternoon"
+msgstr "Jour 1 Après-midi"
+
+#: src/SUMMARY.md:291
+msgid "Day 2 Morning"
+msgstr "Jour 2 Matin"
+
+#: src/SUMMARY.md:292
+msgid "Day 2 Afternoon"
+msgstr "Jour 2 Après-midi"
+
+#: src/SUMMARY.md:293
+msgid "Day 3 Morning"
+msgstr "Jour 3 Matin"
+
+#: src/SUMMARY.md:294
+msgid "Day 3 Afternoon"
+msgstr "Jour 3 Après-midi"
+
+#: src/SUMMARY.md:295
+msgid "Day 4 Morning"
+msgstr "Jour 4 Matin"
+
+#: src/SUMMARY.md:296
+msgid "Bare Metal Rust Morning"
+msgstr "Bare Metal Rust Matin"
+
+#: src/SUMMARY.md:297
+msgid "Bare Metal Rust Afternoon"
+msgstr "Bare Metal Rust Après-midi"
+
+#: src/welcome.md:1
+msgid "# Welcome to Comprehensive Rust 🦀"
+msgstr "# Bienvenue dans Rust Compréhensible 🦀"
```

Comment:
\"Comprehensive Rust\" following on @mgeisler comment about the title being a brand